docs/bookstore/page-0.5/pg0.5-env-setup.md

```diff
@@ -0,0 +1,229 @@
+
+# Environment Setup
+
+![Image](images/image20.png)
+
+In this page, we will be discussing how to set up your environment, and make sure to run up the UI front end and the Book Review Service’s node server.
+
+## **What does the final deliverable for this section look like?**
+
+- You have a running kubernetes (k8s) cluster on your local laptop, with Knative Installed.
+- You have your front end application deployed as kubernetes deployment with port-forwarding to localhost:3000
+- You have your Node.js application deployed as kubernetes deployment with port-forwarding to localhost:8080
+
+We will be fulfilling each requirement with the order above.
+
+## **File Structure**
+
+![Image](images/image21.png)
+
+The code for the sample app is in `docs/code-samples/eventing/bookstore-sample-app`
+
+Under bookstore-sample-app folder, there are 2 folders:
+
+**/solution**: this folder contains all the yaml files, and the code you needed. Check it when you get stuck.
+
+**/start**: this folder contains the necessary files for you to get started. Write all the configuration files yourself following the tutorial!
+
+![Image](images/image16.png)
+
+!!! tip
+
+    Kuack suggests you to start from **/start**, write all the configuration files as you go over the tutorial, and check solutions when you get stuck.
+
+
+## **Shortcut**
+
+![Image](images/image10.png)
+
+Running `docs/code-samples/eventing/bookstore-sample-app/start/setup.sh` will automatically complete all tasks in this section.
+
+!!! warning
+
+    However, if you are not familiar with the process, we recommend reviewing the steps below.
+
+
+
+## **Instructions**
+
+### **Task 1: Set Up a Running Kubernetes Cluster with Knative Installed**
+
+![Image](images/image13.png)
+
+Please follow the instructions here [https://knative.dev/docs/install/](https://knative.dev/docs/install/) to spin up your cluster!
+
+
+
+???+ success "Verify"
+
+    You should see the pods in knative-eventing and knative-serving are running before proceeding forward.
+
+    ![Image](images/image2.png)
+
+#### **Extra Step for Minikube Users:**
+
+![Image](images/image3.png)
+
+Attention! People always forget this! If you're not using the Knative Quick Start, manually set up the tunnel to connect to LoadBalancer services.
```

Comment:
```suggestion
Attention! In case you're not using the Knative Quick Start, set up the tunnel manually to connect to services of type `LoadBalancer`:
```